src/unix/loop.c

```diff
@@ -33,7 +33,7 @@ int uv_loop_init(uv_loop_t* loop) {
   uv__signal_global_once_init();
 
   memset(loop, 0, sizeof(*loop));
-  heap_init((struct heap*) &loop->timer_heap);
+  //heap_init((struct heap*) &loop->timer_heap);
```

Comment:
Just delete it it entirely, the diff will show what you are deleting.

Also, is the heap still used somewhere else? If not, it might be a good chance to delete the source code for the heap.